include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -16,7 +16,7 @@
 #ifndef _ONEDPL_parallel_backend_sycl_radix_sort_H
```

Comment:
Sorry, I don't understand the purpose of this change. If we use the API from the <some> header it should be explicitly included. Standard guarantees the dependency between `<climits>` and `<limits>` in neither direction. I basically don't care about the specific implementation of C++ standard library.

I also don't understand changing API from `<climits>` to `limits`. Is there some analysis why it can be safely changed?  As far as I can see there is not guarantee that `CHAR_BIT` is `unsigned char`.

Reply:
'<climits>' was removed because only 'CHAR_BIT' is used from there.  'CHAR_BIT' можно заменить на 'std::numeric_limits<unsigned char>::digits'. See here https://en.cppreference.com/w/cpp/types/numeric_limits/digits#:~:text=unsigned%20char,CHAR_BIT .  With this in mind, we can exclude '<climits>'.

Reply:
This is a good analysis